packages/tools/src/types/EventTypes.ts

```diff
@@ -104,9 +104,9 @@ type ToolActivatedEventDetail = {
  */
 type AnnotationAddedEventDetail = {
   /** unique id of the viewport */
-  viewportId: string;
+  viewportId?: string;
```

Comment:
Yes, this should have been like this from the beginning, but we accidentally linked it to the Ids :()